api/testdata/specs/npm_tarballs/additional_files.txt

```diff
@@ -52,9 +56,14 @@ export const foo = 'bar';
   "type": "module",
   "dependencies": {},
   "exports": {
-    "./foo": "./foo.js",
-    "./bar": "./bar.json"
+    "./foo": {
+      "types": "./foo.d.ts",
+      "default": "./foo.js"
+    },
+    "./bar": {
+      "default": "./bar.json"
+    }
   },
-  "_jsr_revision": 7
+  "_jsr_revision": 0
```

Comment:
why 0 instead of 8?

Reply:
This is just for the fixture files, so we don't have to update all the fixture files when we update the revision.